app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
-  def self.current_user
-    @user_all_initial = User.first
-  end
+  before_action :authenticate_user!
```

Comment:
- Please make sure your web app enables the registration and login of users:
- 
![Capture](https://user-images.githubusercontent.com/63972670/194421572-dcbf6fd3-96ba-463e-b915-73a64b9f6240.JPG)